test-foundry/TestHeapOrderingInvariant.t.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GNU AGPLv3
```

Comment:
Maybe name it `InvariantHeapOrdering.t.sol` ? That way it is easier to match only invariant contracts (and it is cleaner IMO)

Reply:
I changed from this, because I found that it was a mess in my files explorer. But you raise a good point.

Reply:
Btw, I think that the `Test` prefix everywhere is useless

Reply:
> because I found that it was a mess in my files explorer.

Why do you think that ?

Reply:
now I have 

```
test
  ∟ TestHeapOrdering.t.sol
  ∟ TestHeapOrderingInvariant.t.sol
  ∟ TestThreeHeapOrdering.t.sol
  ∟ TestThreeHeapOrderingInvariant.t.sol
```

But that's a detail, I can change back if you want

Reply:
Ok, let's keep it like that. Maybe one day we will remove the prefix `Test` and all will be good